static_string.h

```diff
@@ -29,20 +38,24 @@ SOFTWARE.
 #ifndef SNW1_STATIC_STRING_H
 #define SNW1_STATIC_STRING_H
 
-#include <array>
 #include <string>
 #include <limits>
 
+#define USE_USER_LITERALS 0
```

Comment:
Я бы написал это в виде
```suggestion
#ifndef USE_USER_LITERALS
#define USE_USER_LITERALS 0
#endif
```
чтобы можно было задавать значение извне без изменения кода